raspberrypi2-cxx11.cmake

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015, Alexandre Pretyman
+# All rights reserved.
+
+if(DEFINED POLLY_RASPBERRYPI2_CXX11_CMAKE)
+  return()
+else()
+  set(POLLY_RASPBERRYPI2_CXX11_CMAKE 1)
+endif()
+
+#Maybe this line should be in polly_init and not in polly_common?
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/utilities")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
```

Comment:
For for in toolchains first we include `polly_init` explicitly, then use it, then include `polly_common` explicitly, then other modules available for include.